src/Serializer/RatingDenormalizer.php

```diff
@@ -18,18 +22,18 @@ public function denormalize($data, $class, $format = null, array $context = []):
     {
         $notice = $context['notice'];
         if (!$notice || !$notice instanceof Notice) {
-            throw new \LogicException('RatingDenormalizer->denormalize must be called with a Notice instance in the context.');
+            throw new LogicException('RatingDenormalizer->denormalize must be called with a Notice instance in the context.');
         }
 
         return new Rating(
             $notice,
             $data['ratingType'],
             new Context(
-                new \DateTime(),
-                isset($data['context']['url']) ? $data['context']['url'] : '',
-                isset($data['context']['geolocation']) ? $data['context']['geolocation'] : ''
+                new DateTime(),
+                isset($data['context']['url']) ? substr($data['context']['url'], 0, 1000) : '',
```

Comment:
Why 1000?

Reply:
It was the same as DB but this was completely unexplicit ! 

Made it better !

Reply:
You're saying that doctrine text without an explicit length woudl default to 1000 chars?
Didn't find these number while browsing the doc.
Anyway if that the case let's explicitly set a limit in the property's mapping and choose one that suit the usecase instead of an arbitrary one.
> http://sqlines.com/mysql/datatypes/varchar

Reply:
It’s already defined here : https://github.com/dis-moi/backend/blob/665264badb7f7e5331d6cf97779fb5de14b156f7/migrations/Version20161214123901.php#L18

Reply:
I know it's a detail but I'm still not convinced. The mapping should be **explicit** about the limit. This info shouldn't come from the database storage.

Reply:
Of course ! I’ve specified the `@Assert\Length` but forgot the length in the `@ORM\Column` !

Updated @lutangar  !

Reply:
Close! :D 

But I was talking about this `length`:
```
@ORM\Column(name="url", type="string", length=1000, nullable=true)
```

Reply:
Sorry ! I forgot to push ! This what I did but referencing the `const` 

Can you please check again @lutangar